torch/csrc/dynamo/compiled_autograd.h

```diff
@@ -174,6 +174,7 @@ struct AutogradCompilerCall {
   std::vector<c10::SafePyObject> hooks;
   NodeCalls node_calls;
   SizeInput::DynType default_dyn_type = SizeInput::STATIC;
+  bool keep_graph = false;
```

Comment:
I don't like exposing this in python, this is not something that eager does and is much slower to read/write than a C++ variable.

Reply:
I don't think you need this anyway?